src/components/taglib/TransformHelper/convertToComponent.js

```diff
@@ -92,7 +99,7 @@ module.exports = function handleComponentBind(options) {
     if (rendererModule) {
         if (rendererModule.inlineId) {
             markoComponentVar = rendererModule.inlineId;
-        } else {
+        } else if (!isImplicitComponent) {
```

Comment:
It's not possible for this to fall into an `else` case right? Otherwise `markoComponentVar` would be `undefined`. Maybe that's intended.

Reply:
It's okay if `markoComponentVar` is undefined. That is handled by `markoComponentVar || builder.literal({})` later in the file.